### PR TITLE
Update the release note with newly added post quantum algorithms in the Crypto module

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
$subject.

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.9.0%22+label%3AType%2FBug).
 
+### New features
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:57]: Should be the first section under "Ballerina library updates"?

Comment (Bhashinee) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:51]: Fixed in [e264ba8](https://github.com/ballerina-platform/ballerina-dev-website/pull/8892/commits/e264ba8234166595d22d42a38ff92cc267238fa9)

**Comment by sm1990** [comment]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -84,6 +84,14 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced GraphQL server-side caching support
 
+#### `crypto` package
+
+- Introduced new APIs for ML-KEM-768(Kyber768) key encapsulation mechanism.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:89]: Unless there's a specific reason not to, shall we add space before (?
```suggestion
- Introduced new APIs for ML-KEM-768 (Kyber768) key encapsulation mechanism.
```

Comment (Bhashinee) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md]: Fixed in [78b1103](https://github.com/ballerina-platform/ballerina-dev-website/pull/8892/commits/78b110378dbf2ec76f4702e6ccfd5df857239bc8)

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**